docs/debugging.md

```diff
@@ -1,4 +1,67 @@
-# Iterating on the installer without deploying the operator
+# Debugging the AWX Operator
+
+## General Debugging
+
+When the operator is deploying AWX, it is running the `installer` role inside the operator container. If the AWX CR's status is `Failed`, is it useful to look at the awx-oeprator container logs, which shows the output of the installer role. To see these logs, run:
```

Comment:
should this be "it is usefule" rather than is it?

Reply:
thx, I reworded it and also fixed a spelling typo on the same line.